src/var/mutation_counting.jl

```diff
@@ -27,128 +60,168 @@ The strictest test of ambiguity, if either nucleotide is not
 A, T, G, or C, then this function returns true.
 """
 @inline function is_ambiguous_strict{T<:Nucleotide}(a::T, b::T)
-    return isambiguous(a) || isambiguous(b)
+    return isambiguous(a) | isambiguous(b)
```

Comment:
I assume this is intentional, in which case perhaps document why bitwise rather than logical?